nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPathNullnessPropagation.java

```diff
@@ -1086,18 +1093,27 @@ private final class ReadableUpdates implements Updates {
 
     @Override
     public void set(LocalVariableNode node, Nullness value) {
-      values.put(AccessPath.fromLocal(node), checkNotNull(value));
+      values.put(AccessPath.fromLocal(node), value);
     }
 
     @Override
     public void set(VariableDeclarationNode node, Nullness value) {
```

Comment:
why did we remove the `checkNotNull`?

Reply:
We are checking NullAway with NullAway now, and `value` is not marked `@Nullable`, so it should always be non-null. I think this is from before NullAway was running on itself 😄